src/reflect/scala/reflect/internal/Symbols.scala

```diff
@@ -1677,7 +1682,12 @@ trait Symbols extends api.Symbols { self: SymbolTable =>
         this.infos = oldest
         oldest
```

Comment:
AFAICT, this optimization is now subsumed by the one you've added. I've removed it in https://github.com/retronym/scala/pull/73

Note: this code is only used in multi-Run compilation (notably presentation compiler and REPL).

Reply:
I have turned this method into a procedure, which returns `Unit`, and just assigns the value to the `infos` mutable variable.